tools/psa/tfm/bin_utils/assemble.py

```diff
@@ -3,6 +3,8 @@
 # Copyright 2017 Linaro Limited
 # Copyright (c) 2017-2019, Arm Limited.
 #
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
100644 → 100755 - can you revert the permission change for this and file below

Reply:
755 (executable bit) is the latest permission in the upstream project: https://github.com/ARMmbed/trusted-firmware-m/tree/mbed-tfm-1.2/bl2/ext/mcuboot/scripts
and those files are imported. Can we keep them as they are?

Reply:
OK, interesting  they changed it - a mistake I guess :)

Reply:
Yea it was changed between TF-M v1.1 -> v1.2